frontend/packages/ceph-storage-plugin/src/utils/install.ts

```diff
@@ -52,3 +57,14 @@ export const getAssociatedNodes = (pvs: K8sResourceKind[]): string[] => {
 
   return Array.from(nodes);
 };
+
+export const shouldDeployMinimally = (nodes: NodeKind[]) =>
+  nodes.reduce((acc, curr) => {
+    const cpus = humanizeCpuCores(getNodeCPUCapacity(curr)).value;
+    const memoryRaw = getNodeAllocatableMemory(curr);
+    const memory = humanizeBinaryBytes(convertToBaseValue(memoryRaw)).value;
+    if (memory < 60 && cpus < 16) {
```

Comment:
Shouldn't this be OR condition?
```suggestion
    if (memory < 60 ||  cpus < 16) {
```